src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1677,4 +1677,36 @@ Please specify one of them using the --source option to proceed.</value>
     <value>{0} package(s) have pins that prevent upgrade. Use the 'winget pin' command to view and edit pins. Using the --include-pinned argument may show more results.</value>
     <comment>{Locked="winget pin","--include-pinned"} {0} is a placeholder replaced by an integer number of packages</comment>
   </data>
+  <data name="WindowsFeatureNotFoundOverride" xml:space="preserve">
+    <value>The following Windows Feature(s) were not found; proceeding due to --ignore-missing-dependencies.</value>
+    <comment>"Windows Feature(s)" is the name of the options Windows features setting.
+{Locked="--force"}
+{Locked="--ignore-missing-dependencies"}</comment>
+  </data>
+  <data name="WindowsFeatureNotFoundOverrideRequired" xml:space="preserve">
+    <value>The following Windows Feature(s) were not found. To proceed with installation, use --ignore-missing-dependencies.</value>
+    <comment>"Windows Feature(s)" is the name of the options Windows features setting.
+{Locked="--force"}
+{Locked="--ignore-missing-dependencies"}</comment>
+  </data>
+  <data name="IgnoreMissingDependenciesArgumentDescription" xml:space="preserve">
+    <value>Ignores missing Windows Feature dependencies</value>
+    <comment>"Windows Feature" is the name of the options Windows features setting.</comment>
+  </data>
+  <data name="EnablingWindowsFeatures" xml:space="preserve">
+    <value>Enabling Windows Feature dependencies...</value>

```

Comment:
As a user, I would like to know specifically which features are being enabled
```
    <value>Enabling Windows Feature {0}.</value>
```

Reply:
There is a symbolic name and a friendly (localized) name.  I don't know how easy it is to get the friendly name from the symbolic name.  Which do you expect to see (or both)?

Reply:
Changed to showing friendly name as well as the symbolic name such as `[netfx3]`